configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py

```diff
@@ -0,0 +1,192 @@
+    dict(type='RawFrameDecode'),
+    dict(type='Resize', scale=(340, 256), keep_ratio=True),
+    dict(type='CenterCrop', crop_size=224),
+    dict(type='Flip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='FormatShape', input_format='NPTCHW'),
+    dict(
+        type='Collect',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ],
+        meta_keys=[]),
+    dict(
+        type='ToTensor',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ])
+]
+test_pipeline = [
+    dict(
+        type='SampleProposalFrames',
+        clip_len=1,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5,
+        mode='test'),
+    dict(type='RawFrameDecode'),
+    dict(type='Resize', scale=(340, 256), keep_ratio=True),
+    dict(type='CenterCrop', crop_size=224),
+    dict(type='Flip', flip_ratio=0),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='FormatShape', input_format='NCHW'),
+    dict(
+        type='Collect',
+        keys=[
+            'imgs', 'relative_proposal_list', 'scale_factor_list',
+            'proposal_tick_list', 'reg_norm_consts'
+        ],
+        meta_keys=[]),
+    dict(
+        type='ToTensor',
+        keys=[
+            'imgs', 'relative_proposal_list', 'scale_factor_list',
+            'proposal_tick_list', 'reg_norm_consts'
+        ])
+]
+data = dict(
+    videos_per_gpu=2,
+    workers_per_gpu=2,
+    train=dict(
+        type=dataset_type,
+        ann_file=ann_file_train,
+        data_prefix=data_root,
+        train_cfg=train_cfg,
+        test_cfg=test_cfg,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5,
+        test_mode=False,
+        verbose=True,
+        pipeline=train_pipeline),
+    val=dict(
+        type=dataset_type,
+        ann_file=ann_file_val,
+        data_prefix=data_root,
+        train_cfg=train_cfg,
+        test_cfg=test_cfg,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5,
+        test_mode=False,
+        pipeline=train_pipeline),
+    test=dict(
+        type=dataset_type,
+        ann_file=ann_file_test,
+        data_prefix=data_root,
+        train_cfg=train_cfg,
+        test_cfg=test_cfg,
+        aug_ratio=0.5,
+        test_mode=True,
+        pipeline=test_pipeline))
+# optimizer
+optimizer = dict(
+    type='SGD', lr=0.001, momentum=0.9,
+    weight_decay=1e-6)  # this lr is used for 8 gpus
+optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
+# learning policy
+lr_config = dict(policy='step', step=[200, 400])
+checkpoint_config = dict(interval=5)
+log_config = dict(interval=1, hooks=[dict(type='TextLoggerHook')])
+# runtime settings
+total_epochs = 1
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+work_dir = './work_dirs/ssn_bninception_450e_thumos14_rgb'
```

Comment:
ssn_r50_1x5_450e_thumos14_rgb_train

Reply:
Since SSN uses different STPP at training and testing, we have to use two configs. But those two configs can share the same checkpoint in same working directory './work_dirs/ssn_r50_450e_thumos14_rgb'.

Reply:
bninception -> r50